Assets/Scripts/Utility/MacroHelper.cs

```diff
@@ -453,6 +453,11 @@ public static void ExpandMacros(ref TextFile.Token[] tokens, IMacroContextProvid
 
                             // Iterate from macro end
                             currentPos = endPos;
+
+                            // "Eating" the | terminator (for postfixes purpose)
+                            // Example: '%di|ern' => 'southern'
+                            if (currentPos < tokenText.Length && tokenText[currentPos] == '|')
```

Comment:
I'm not sure such a pattern would play well with localization. I'm also not keen on changing macro evaluation at this stage - what if someone wanted to make a text-based table (ex: '|%di|...|...|' -> '|south|...|...|). A bit contrived, but with a macro of fixed size, it would make more sense

Reply:
That was added specifically to enhance localization. We need postfixes in other languages. 
And I don't think this table (`|%di|...|...|`) would eve exist, honestly :D

Reply:
Or by that time, they would fiure out something